Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM golang:1.14 AS builder
```

Comment:
I'd rather use golang 1.16 or at least 1.15.

1.14 recently reached EOL and it doesn't make sense to use it in a dockerfile (IMHO).

Reply:
Done